deep_agent/src/memory/consolidation.py

```
@@ -137,13 +137,19 @@ async def consolidate_user_memories(
+                    (str(memories[idx]["id"]), user_id),
                 )
                 deleted += 1
 
         if deleted:
             await conn.commit()
+            try:
+                from deep_agent.src.cache.personalization_cache import invalidate
+
+                await invalidate(user_id)
+            except Exception:
+                pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)